types/barnard59-base/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Tomasz Pluskiewicz <https://github.com/tpluscode>
 //                 Benjamin Hofstetter <https://github.com/BenjaminHofstetter>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 4.4
+// TypeScript Version: 4.9
```

Comment:
Why do you need these TS version changes?

Reply:
Otherwise tests will fail

Reply:
That is because you updated the TS version in `types/readable-stream` from `2.3` to `4.9`. Why do you need this? Which 4.9 feature do you use?

Reply:
I had to add it for the `compose` change

Reply:
and because now the [test uses](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65929/files#diff-9b47492a59fb224450d04216ed4d3e065d2725a3c62f9c93aa80dcbf5ba49bf1R3) `tap`, `dot`, and `spec` which are new types.

Reply:
as mentioned on this example from [Node doc](https://nodejs.org/api/test.html#runoptions)

Reply:
Could you please add a separate commit with reverting TS version changes? Will see what the error is.

Reply:
[d11b8ae](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65929/commits/d11b8ae44f4327aebad5fc2ecb488c1ac32fe9f8)

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/5656235492/job/15322862921?pr=65929#step:6:3497

Reply:
@rozzilla Ah, I know why. You have not added changes to `types/node/ts4.8/stream.d.ts`